getFeatureFlagById/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"log"
+	"net/http"
+)
+
+func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	id, _ := req.PathParameters["flagId"]
+	
+	featureFlag, err := database.ProcessGetFeatureFlagByHashKey(utils.Id, id)
+	if err != nil {
+		return utils.ServerError(err)
+	}
+
+	if featureFlag == nil {
+		log.Println("Feature Flag not found")
+		return utils.ClientError(http.StatusNotFound, "Feature flag not found")
+		
+	}
+	log.Println(featureFlag, " is the feature flag")
+	
+	//marshal the item to JSON
```

Comment:
the code is self explanatory do we need this comment?

Reply:
No